PWGCF/FEMTOSCOPY/FemtoDream/AliFemtoDreamTrackMCHist.cxx

```diff
@@ -76,7 +76,9 @@ AliFemtoDreamTrackMCHist::~AliFemtoDreamTrackMCHist() {
 AliFemtoDreamTrackMCHist::AliFemtoDreamTrackMCHist(bool contribSplitting,
                                                    bool DCADist,
                                                    bool DoMultBinning,
-                                                   bool checkMother)
+                                                   bool checkMother,
+						   float fpTmin,
+						   float fpTmax)
     : fpTmin(0.5),
       fpTmax(4.05),
       fpTbins(20),
```

Comment:
Same thing as for the TrackHists - you also don't use the newly assigned values to initialize the histograms. Is that what you want?